website/client/components/groups/chat.vue

```diff
@@ -233,6 +284,84 @@
       display: inline-block;
       width: 100px;
     }
+
+    .chat-count {
+      margin-top: 8px;
+      display: block;
+    }
+  }
+
+  .v-tribute {
+    width: 100%;
+  }
+</style>
+
+<style lang="scss">
+  @import '~client/assets/scss/tiers.scss';
```

Comment:
it looks like this file is being imported multiple times acrsso the website, resulting in duplicate css code, can you remove all the `@import '~client/assets/scss/tiers.scss';` (there are a couple more) and import it only once in `assets/scss/index`